### PR TITLE
User Group Dim 2

### PR DESCRIPTION
**Pull request by benrudolph**

### CODE REVIEW

**Comment by benrudolph** [comment]
TRAVIS!

File: corehq/warehouse/transforms/sql/user_group_dim.sql [hunk 1]
```diff
@@ -13,31 +23,18 @@ SELECT
...
-    SELECT
-        UNNEST(removed_user_ids) AS removed_user_id,
-        group_id AS group_id,
-        true AS deleted
+        group_id AS group_id
     FROM
         {{ group_staging }}
-)
-) combined_user_group_table
+    WHERE doc_type = 'Group'
```
Comment (gcapalbo) [inline corehq/warehouse/transforms/sql/user_group_dim.sql]: i don't think the subquery is needed - looks like we can join normally and then unnest() in the main select

Comment (gcapalbo) [inline corehq/warehouse/transforms/sql/user_group_dim.sql]: (never mind, my idea won't work, fine to leave as is)

Comment (benrudolph) [inline corehq/warehouse/transforms/sql/user_group_dim.sql]: (offlined, going to leave as is for now and address later with intermediate table)

**Review by gcapalbo — APPROVED** [review]

**Pull request merged**